### DISCUSSION TITLE
Other than stripe

### DISCUSSION THREAD

**Discussion by diamant3** [community/community#33322] [Other Feature Feedback, Questions, & Ideas] [answered]
is there anything i can use other than stripe? thanks.

**Comment by cheshire137** [comment, marked as answer]
Yes! There's the option of using a fiscal host to receive sponsorship payments. See https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/using-a-fiscal-host-to-receive-github-sponsors-payouts.
